src/YoutubeApi.cpp

```diff
@@ -83,6 +83,14 @@ bool YoutubeApi::getChannelStatistics(String channelId){
 
 	return getChannelStatistics(tempStr);
 }
+bool YoutubeApi::getVideoStatistics(String videoId){
+
+	int strLen = videoId.length() + 1; 
+	char tempStr[strLen];
+	videoId.toCharArray(tempStr, strLen);
+
+	return getVideoStatistics(tempStr);
```

Comment:
You should be able to save yourself some hassle here by using `videoId.c_str();`

https://www.arduino.cc/reference/en/language/variables/data-types/string/functions/c_str/